components/recurring-contributions/UpdateOrderPopUp.js

```diff
@@ -135,8 +147,9 @@ const getTierAmountOptions = (selectedTier, contribution) => {
   if (selectedTier && !selectedTier?.flexible) {
     return [buildOptionFromAmount(selectedTier.amount)];
   } else {
-    const presets = selectedTier?.presets || [500, 1000, 2000, 5000];
-    return [...presets.map(buildOptionFromAmount), { label: 'Other', value: 100 }];
+    const presets = selectedTier?.presets || DEFAULT_PRESETS;
```

Comment:
Using `getTierPresets` from `lib/tier-utils.js` could be a good way to make sure `minAmount` is properly accounted for

```es6
const presets = getTierPresets(selectedTier, collectiveType);
```

Reply:
We can do it but it's a bit complicated to get the `collectiveType` and pass it all the way.

Added a TODO there:
https://github.com/opencollective/opencollective-frontend/pull/6374/commits/7ecdd8e0f4ce26eb9741b7c4651dfaf18cfed6e5